lib/parsers/document_parser.js

```diff
@@ -305,6 +303,26 @@ function fillAssociations() {
   }
 }
 
+function convertAnnotationsToOptions(annotations) {
+  const result = {};
+  annotations.forEach(o => {
+    const value = o.method ? o.method : true;
+    if (o.option in result) {
+      const oldValue = result[o.option];
+      if (Array.isArray(oldValue)) {
+        if (value !== oldValue) {
```

Comment:
I'm not sure it works.
Here, if I'm getting this right, `oldValue` is an array, right? so comparing an array if another won't work (try [1] === [1] for instance)

Reply:
Je sais bien, je pense on relisant que je cherchais à faire ça:
`if (!oldValue.includes(value)) {` j'étais décidément très fatigué, merci d'avoir vu.